### PR TITLE
landing page done

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
1. Added Navbar
2. Added Footer
3. Homepage Heading and description
4. Homepage Image
Screenshot 2022-06-04 102839

### CODE REVIEW

**Review by arpit01923 — APPROVED** [review]
- clean code
- good folder structure
- good to merge

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,15 @@
...
+    --white-clr:white;
+    --primary-color:#EF4444;
+    --nav-bg-color :#112457;
+    --prime-blue:#93C5FD;
+    --secondary-pink:#F9ABD4;
+    --info-clr:#A5B4FC;
+    --warning-clr:#FCD34D;
+    --danger-clr:#DC2626;
+    --success-clr:#10B981;
+    --grey-clr:grey;
```
Comment (SanketDhabarde) [inline src/App.css:13]: you can give this in hex as well just to be consistent

**Review by SanketDhabarde — APPROVED** [review]
- Looks good 
- Code structure is good
- UI is looking good

File: src/App.css (hunk 1 above)
Comment (govardhanreddyc79) [reply src/App.css:13]: sure

**Pull request merged**